ckan/tests/controllers/test_feed.py

```diff
@@ -81,3 +84,56 @@ def test_custom_atom_feed_works(self):
         assert '<title>{0}</title>'.format(dataset1['title']) in res.body
 
         assert '<title>{0}</title>'.format(dataset2['title']) not in res.body
+
+
+class TestFeedInterface(helpers.FunctionalTestBase):
+    @classmethod
+    def setup_class(cls):
+        super(TestFeedInterface, cls).setup_class()
+        plugins.load('test_feed_plugin')
```

Comment:
Do 

```python
    if not plugins.plugin_loaded('test_feed_plugin'):
        plugins.load('test_feed_plugin')
```

This prevents problems when tests fail and plugins are not unloaded properly